contracts/RegisterAuth.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import "./ISSVNetwork.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+interface IRegisterAuth {
+    struct Authorization {
+        bool registerOperator;
+        bool registerValidator;
+    }
+
+    function setAuth(address userAddress, Authorization calldata auth) external;
+
+    function getAuth(address userAddress) external view returns (Authorization memory);
```

Comment:
@lior-blox I keep get function to allow calls like `registerAuth.getAuth(msg.sender).registerOperator` instead of `(bool registerOperator, bool registerValidator) = registerAuth.getAuth(msg.sender)` from `SSVNetwork`